### DISCUSSION TITLE
How to add new platforms and import transaction data?

### DISCUSSION THREAD

**Discussion by Tky-Phoenix** [ghostfolio/ghostfolio#950] [Q&A]
I am trying to get Ghostfolio set up but I can't figure out how to add new platforms and import my historic data.
I was looking for some documentation but couldn't find any. I'm sure there is some but I've been looking at the screen the whole day and just can't find it.
If someone could please point me in the right direction to get started, that would be great.
Thanks!

**Comment by dtslvr** [comment]
Hi @Tky-Phoenix

Please check these discussions:
* https://github.com/ghostfolio/ghostfolio/discussions/735
* https://github.com/ghostfolio/ghostfolio/discussions/915

**Reply by Tky-Phoenix** [reply]
@dtslvr Thank you for the prompt response.

For the platform one, I am too much of a novice to figure out how to do that. Will have to consult with a friend of mine.
For the historical transactions, does any .json or .csv file work as long as it has the same categories?
Date/type/symbol/quantity/unit price/fee/value/currency/account
